tests/Browser/BlogManageTest.php

```diff
@@ -0,0 +1,562 @@
+     * case 1.1: Test screen Blog list
+     *
+     * @return void
+     */
+    public function testLoginBlog()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->assertPathIs($data['path']['blog']);
+        });
+    }
+
+    /**
+     * Test popup create blog with user can see blog.
+     * case 1.2: Test popup Create blog
+     *
+     * @return void
+     */
+    public function testShowPopupCreateBlog()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->press('Create Blog')
+                    ->waitFor('#createModal')
+                    ->assertPresent('#createModal');
+        });
+    }
+
+     /**
+     * Test screen Edit blog by user.
+     * case 1.3: Edit blog: Test screen Edit blog by user
+     *
+     * @return void
+     */
+    public function testScreenEditBlog()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->click('#blog-edit-2')
+                    ->assertPathIs('/blog/2/edit');
+        });
+    }
+
+    /**
+     * Test screen delete blog by user.
+     * case 1.4: Delete blog: Test screen delete blog by user
+     *
+     * @return void
+     */
+    public function testScreenDeleteBlog()
+    {
+        $user = $this->userCanSeeAndDelete();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->click('#blog-delete-5')
+                    ->waitFor('#deleteModal5')
+                    ->assertSee('Delete this blog?');
+        });
+    }
+
+    /**
+     * Blog list have not permission see blog
+     * case 1.5: Blog list have not permission see blog
+     *
+     * @return void
+     */
+    public function testBlogListUseHaveNotPermissionSeeBlog()
+    {
+        $user = $this->userCannotSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->assertPathIsNot($data['path']['blog']);
```

Comment:
Expected in Test Case report not match this assert , what is the problem ?

Reply:
I fixed